dietpi-install.sh

```diff
@@ -54,10 +54,23 @@ else
 fi
 
 qm set "$ID" --name 'dietpi' >/dev/null
-qm set "$ID" --description '### [DietPi Website](https://dietpi.com/)
-### [DietPi Docs](https://dietpi.com/docs/)  
-### [DietPi Forum](https://dietpi.com/forum/)
-### [DietPi Blog](https://dietpi.com/blog/)' >/dev/null
+
+# Description with logo and horizontal links
+DESCRIPTION='
+<p align="center">
+  <img src="https://i.ibb.co/rH7GPX5/dietpi.png" alt="DietPi Logo" width="40" />
+  <br/>
+  <strong>DietPi VM</strong>
+  <br/>
+  <a href="https://dietpi.com/">DietPi Website</a> &bull; 
+  <a href="https://dietpi.com/docs/">DietPi Docs</a> &bull; 
+  <a href="https://dietpi.com/forum/">DietPi Forum</a>
+  <br/>
+  <a href="https://dietpi.com/blog/">DietPi Blog</a>
```

Comment:
```suggestion
  <img src="https://dietpi.com/images/dietpi-logo_128x128.png" alt="DietPi Logo" width="40">
  <br>
  <strong>DietPi VM</strong>
  <br>
  <a href="https://dietpi.com/">Website</a> &bull; 
  <a href="https://dietpi.com/docs/">Documentation</a> &bull; 
  <a href="https://dietpi.com/forum/">Forum</a>
  <br>
  <a href="https://dietpi.com/blog/">Blog</a> &bull; 
  <a href="https://github.com/MichaIng/DietPi">GitHub</a>
```
Since it's clear that it's about DietPi, skipping the "DietPi" prefix on every link text should be fine. Also adding GitHub, which aligns it to the common set of links shown on each of this pages.

Reply:
Ah yes that looks better, thanks!